cmd/logger/console.go

```diff
@@ -27,7 +27,19 @@ import (
 // in plain or json format to the standard output.
 type ConsoleTarget struct{}
 
-func (c *ConsoleTarget) send(entry logEntry) error {
+func (c *ConsoleTarget) send(e interface{}) error {
+	entry, ok := e.(logEntry)
```

Comment:
Should we not be printing on Console?

Reply:
We don't need @kannappanr - this is only needed for HTTP target.